elements/timeline-grid.html

```diff
@@ -2,14 +2,28 @@
 <link rel="import" href="timeline-element.html">
 <link rel="import" href="../bower_components/iron-flex-layout/iron-flex-layout-classes.html">
 
-<dom-module id="timeline-grid">
+<dom-module id="timeline-grid" current-count="{{currentCount}}">
```

Comment:
This is the binding for the current count property. This allows socket.io to pass the current count as needed, at which point the property is updated and Polymer can trigger its property changed event.